### ISSUE TITLE
Miss @endo/init package

### ISSUE THREAD

**Issue by kennyrowe** [Agoric/agoric-sdk#4436]
## Describe the bug
When building the SDK with a mac M1 the final build step fails

## To Reproduce
1. Using Node 14.15
2. Followed instructions for building with apple silicon (ie, install canvas)
3. `yarn  install` 
4. `yarn build`
5.  error
```
internal/process/esm_loader.js:74
    internalBinding('errors').triggerUncaughtException(
                              ^
Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@endo/init'
```

## Platform Environment
 - Node version 14.15
 - Macbook pro M1 max chip

## Additional context
Tried adding package with `yarn add @endo/init` and got

```
yarn add v1.22.15
error Running this command will add the dependency to the workspace root rather than the workspace itself, which might not be what you want - if you really meant it, make it explicit by running this command again with the -W flag (or --ignore-workspace-root-check).
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

**Comment by dckc** [comment]
@kennyrowe found a work-around: use the x86 emulator feature of the terminal app.

**Comment by mhofman** [comment]
> 1. Using Node 14.15

Node 14 is not supported on M1

**Comment by dckc** [comment]
> Node 14 is not supported on M1

where do we / should we document that, @mhofman ?

**Comment by mhofman** [comment]
It's a Node.js limitation, not an agoric-sdk limitation.

**Comment by dckc** [comment]
@kennyrowe try with node 16 and without the x86 emulator, please?

**Comment by tgrecojs** [comment]
> Node 14 is not supported on M1

Node v14 has worked fine when building the sdk on an m1 machine.  I'm running v14.17.6 but i can confirm it worked with other v14 versions as well.

I took a minute to reproduce installing and building from a new repo and can confirm that everything succeeded without issue.

```
tg@Thomass-Mac-mini-2 ~[agoric-sdk-repro] % node -v
v14.17.6
tg@Thomass-Mac-mini-2 ~[agoric-sdk-repro] % agoric --version
0.13.21
tg@Thomass-Mac-mini-2 ~[agoric-sdk-repro] % yarn -V
yarn install v1.22.15
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
success Already up-to-date.
$ patch-package
patch-package 6.2.2
Applying patches...
@lerna/conventional-commits@3.18.5 ✔
@lit/reactive-element@1.0.1 ✔
acorn@7.4.1 ✔
depd@1.1.2 ✔
express@4.17.1 ✔
external-editor@3.1.0 ✔
lit@2.0.2 ✔
lit-element@3.0.1 ✔
lit-html@2.0.1 ✔
node-fetch@2.6.7 ✔
✨  Done in 0.61s.
```

**Comment by tgrecojs** [comment]
seeing this issue now #4460 which clarifies agoric-sdk's support for node v14 🤦‍♂️ not sure if it's specific to docker but curious as to whether this is still valid as an open issue, better suited as a documentation task.... 🤷‍♂️ hope its useful feedback nonetheless 🙂

**Comment by mhofman** [comment]
> seeing this issue now #4460 which clarifies agoric-sdk's support for node v14 🤦‍♂️ not sure if it's specific to docker but curious as to whether this is still valid as an open issue, better suited as a documentation task.... 🤷‍♂️ hope its useful feedback nonetheless 🙂

I'm not sure I follow. #4460 was because #4420, which was merged after this issue was opened, introduced a dev dependency that has a higher node version requirement, so we bumped up our CI image.

I'm not sure which arch or from where you installed node.js, but the official package for Node 14 does not support the M1 architecture, only the x86 emulation. That is nothing specific to this repo.

The TLDR is that we still support the latest Node 14 and 16 LTS, for platforms which are officially supported. I agree we should probably remove the `.15` minor specifier from documentations, and simply refer to latest LTS.
